src/style.css

```diff
@@ -144,6 +144,10 @@ div.task-container ul li div {
   border-bottom: 1px solid #b8b8b8;
 }
 
+.line-through {
+  text-decoration: line-through;
+}
+
```

Comment:
## OPTIONAL  ♻️

- [ ] Add the property `cursor: pointer` to the icons, this will help with the UI. 
😄 🔧

Reply:
Fixed and thanks for your review